modules/calendar/full_calendar/fullcalendar.css

```diff
@@ -0,0 +1,1413 @@
+/*!
```

Comment:
I'll look at this later. I haven't yet started with color picker but it is on my priority list.